src/sql/workbench/parts/editData/browser/editData.component.ts

```diff
@@ -510,7 +510,7 @@ export class EditDataComponent extends GridParentComponent implements OnInit, On
 					? self.rowIdMappings[self.currentCell.row]
 					: self.currentCell.row;
 
-				return self.dataService.updateCell(sessionRowId, self.currentCell.column - 1, self.currentEditCellValue);
+				return self.dataService.updateCell(sessionRowId, self.currentCell.column - 1, self.currentEditCellValue.replace('\u0000', String.fromCharCode(13)));
```

Comment:
What if the line break was \r\n or \n? This is still going to modify the value in that case right?

Reply:
Yea, I thought about that too, honestly I don't know when the new line is coded as either /r/n or /n. That's why I used the enter key code that gives the new line character code irrespective. At the end the user entered a line break and he wants that preserved. And am up for suggestions to make it cleaner.

Reply:
We shouldn't be modifying the content regardless. So if you aren't going to fix that issue here then please open another issue to later resolve that.